src/Symfony/Bridge/PhpUnit/Legacy/CoverageListenerForV6.php

```diff
@@ -22,8 +23,10 @@
  *
  * @internal
  */
-class CoverageListenerForV6 extends BaseTestListener
+class CoverageListenerForV6 implements TestListener
 {
+    use TestListenerDefaultImplementation;
```

Comment:
the trait has been added to v6.5, isn't it?

Reply:
The trait was added in v6.4.

Initial commit:
https://github.com/sebastianbergmann/phpunit/commit/9d5924f4bf4d4f41529da5e2c98c1d8ed9495e00#diff-f014f0750e1e6f18c6598f8c01d788a5

Parent commit:
https://github.com/sebastianbergmann/phpunit/commit/169bb8351039018b315438d7661606ed5e8609f3

Should I update the `composer.json` for v6?

Reply:
yes, would make sense to me.

Reply:
Added the conflicting versions for v6.